docs/pages/admin-guides/access-controls/rbac/resource-rbac.mdx

```diff
@@ -0,0 +1,108 @@
+
+---
+title: Getting Started With Per-Resource Role-Based Access Control
+description: Implementing fine-grained access control with RBAC.
+---
+
+
+## What is per-resource RBAC?
+
+This granular approach allows you to grant more precise management of user permissions.
+
+Unlike broader, traditional role-based permissions, which can apply to all resources within a namespace or across an entire infrastructure, per-resource RBAC enables administrators to control user access on a per-resource basis. 
+This feature is especially valuable in complex environments with diverse user groups, where some users need restricted access to sensitive systems while others may require broad access.
+
+## Use cases
+
+### Multi-tenant environments
+
+In a multi-tenant setup, per-resource RBAC can ensure that each client or team has access only to their specific resources, preventing any accidental or intentional cross-tenant access.
+
+### Compliance requirements
+For industries with strict regulatory requirements (e.g., finance, healthcare), per-resource RBAC helps in implementing and demonstrating compliance with access control policies.
+
+### Team management
+For new team members or roles, per-resource RBAC facilitates a gradual increase in access privileges as users become more familiar with the systems and demonstrate need and competence.
+When working with external contractors, per-resource RBAC allows granting temporary, limited access to specific resources without exposing the entire infrastructure.
+
+### DevOps workflows
+In CI/CD pipelines, different stages may require different levels of access; per-resource RBAC can be used to grant just-in-time access for specific tasks in the pipeline.
+
+Let's look at some examples of how per-resource RBAC can be implemented:
+
+**SSH access control**
+
+```code
+kind: role
+version: v3
+metadata:
+ name: dev-team
+spec:
+ allow:
+    logins: ['ubuntu']
+    node_labels:
+      environment: ['dev', 'staging']
+    rules:
+      - resources: ['session']
+        verbs: ['read', 'list']
+```
```

Comment:
Is this guide referring to Kubernetes per-resource RBAC, or Teleport roles in relation to infrastructure resources in general?